docs/index.rst

```diff
@@ -89,6 +89,11 @@ look for these files.
     **stsynphot**. If you plan to use the latter anyway, please also read the
     instructions in its documentation.
 
+To install local data files via HTTP into ``/my/local/dir/cdbs``::
```

Comment:
This does not seem relevant to the notebook?

Reply:
Ah, yeah, that's just me having some git confusion with fetching and branches...

Reply:
Still need to get rid of the changes to this file in this PR. Lemme know if you need help with that!